drivers/led/Kconfig.gpio

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2019 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config LED_GPIO
+	bool "GPIO LED driver"
+	depends on GPIO
+	help
+	  This option enables support for the LEDs connected to GPIO
+	  outputs. To be useful the particular board must have LEDs
+	  and they must be connected to the GPIO lines.
+
+if LED_GPIO
+
+config LED_SOFTWARE_BLINK
+	bool "Software blink"
+	help
+	 Support for led software blink based on
```

Comment:
documentation and help text should capitalize ``LED``

Reply:
Ok. I think that I changed it in all places.